.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,83 @@
+name: CI/CD
+
+on:
+  workflow_dispatch: # Allows you to run this workflow manually from the Actions tab
+  pull_request: # Runs whenever a pull request is created or updated
+  push: # Runs whenever a commit is pushed to the repository
+    branches: [master]
+
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+permissions:
+  contents: write # publish a GitHub release
+  pages: write # deploy to GitHub Pages
+  issues: write # comment on released issues
+  pull-requests: write # comment on released pull requests
+
+jobs:
+  sync-s3:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: read
+
+    env:
+      LEGACY_ACCOUNT_OIDC_ROLE: ${{ secrets.LEGACY_ACCOUNT_OIDC_ROLE }}
+      AWS_REGION: ${{ secrets.AWS_REGION }}
+      S3_BUCKET_NAME: ${{ secrets.S3_BUCKET_NAME }}
+      TRIGGER_DEPLOY: ${{ startsWith(github.ref, 'refs/heads/master') }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: wagoid/commitlint-github-action@v5
+        if: github.event_name == 'pull_request'
```

Comment:
Since this repo is primarily managed outside the engineering department, I'd remove this check.
```suggestion
```